cupy/_core/_routines_math.pyx

```diff
@@ -1071,15 +1071,21 @@ _floor_divide = create_ufunc(
 
     ''')
 
-
 _remainder = create_ufunc(
     'cupy_remainder',
     ('bb->b', 'BB->B', 'hh->h', 'HH->H', 'ii->i', 'II->I', 'll->l', 'LL->L',
      'qq->q', 'QQ->Q',
-     ('ee->e', 'out0 = in0 - _floor_divide(in0, in1) * in1'),
-     ('ff->f', 'out0 = in0 - _floor_divide(in0, in1) * in1'),
-     ('dd->d', 'out0 = in0 - _floor_divide(in0, in1) * in1')),
-    'out0 = (in0 - _floor_divide(in0, in1) * in1) * (in1 != 0)',
+     ('ee->e',
+      'out0 = (in1 == INFINITY) ? in0 : '
```

Comment:
The identifier "INFINITY" is undefined. Could you please fix the CUDA compilation error?

Reply:
Can you please suggest some resources which might help me. I have tried doing it in various ways but still getting errors.